ssl/ssl_mcnf.c

```diff
@@ -60,6 +64,7 @@ static int ssl_do_config(SSL *s, SSL_CTX *ctx, const char *name, int system)
         SSL_CONF_CTX_set_ssl_ctx(cctx, ctx);
         libctx = ctx->libctx;
     }
+    conf_diagnostics = OSSL_LIB_CTX_get_conf_diagnostics(libctx);
```

Comment:
if `conf_diagnostics` is true should we set the `SSL_CONF_FLAG_SHOW_ERRORS` flag on `cctx`?

Reply:
Yeah, good point.